src/CheckoutSdk/Payments/Links/IPaymentLinksClient.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace Checkout.Payments.Links
+{
+
+	public interface IPaymentLinksClient
+	{
+		Task<PaymentLinkDetailsResponse> GetAsync(string reference, CancellationToken cancellationToken = default);
```

Comment:
Please remove the `Async` from method names. It should be like that in the Java SDK but here we decided to avoid it. I know it's a good practice, but here is kind of redundant because everything is async :)